src/main/java/hydroguard/api/mock/controllers/account/AccountController.java

```diff
@@ -0,0 +1,153 @@
+package hydroguard.api.mock.controllers.account;
+
+import hydroguard.api.mock.models.account.AccountDTO;
+import hydroguard.api.mock.models.account.AccountsDTO;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/accounts")
+public class AccountController {
+
+    private final List<AccountDTO> accounts = new ArrayList<>();
+
+    @GetMapping
+    public AccountsDTO getAllAccounts(
```

Comment:
Get all voor accounts is niet voor normale users of heeft te veel info, denk je niet? Want waarom zou ik van iemand anders kunnen zien wat hun phone number is? In principe wil je hier alleen terughebben: Id, Username en avatarUrl. Misschien nog email, maar zou niet helemaal weten waarom.